packages/terra-image/src/variants/_DecorativeImage.jsx

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
```

Comment:
About 95% of the code in DecorativeImage.jsx is the same as in [Image.jsx](packages/terra-image/src/Image.jsx). Could we create a `BaseImage` component where these other types of images can inherit from? This would make maintenance easier especially if we plan to support additional types of images.

Reply:
So this would add one more layer on image like below 
```html
  <Image
      alt=" "
      fit="fill"
      isFluid={false}
      src=""
      variant="default"
    >
      <img
        alt=" "
        className="image fill default orion-fusion-theme"
        onError={[Function]}
        onLoad={[Function]}
        src=""
      />
    </Image>
```
Due to this additional wrapper Jest tests of terra-image and terra-profile-image needs to be updated. since most of them are using [wrapper property to validate props ](https://github.com/cerner/terra-core/blob/dc9a5fb21ca05338e25e08b7b1ae524d1a422901/packages/terra-image/tests/jest/Image.test.jsx#L23-L25) . So since this change has larger impact than it is expected should we do it with this Jira..??

Reply:
I feel that change is not critical and it's OK for now to merge as-is, but then turn around and combine the code soon after. As long as the change is made soon I would be OK to approve it as-is.

Reply:
I agree to keep it as is for now. There is nothing wrong with how it is now. Having a base image is just less code so we could even just leave it this way until when there is a need to support additional image types.